src/features/rewards/mobile/mainToggleMobile/style.ts

```diff
@@ -30,26 +30,27 @@ export const StyledLeft = styled<{}, 'div'>('div')`
 `
 
 export const StyledRight = styled<{}, 'div'>('div')`
-  padding: 0px 15px;
-  display: flex;
   height: 66px;
-  align-items: center;
-  margin-top: -3px;
-
-  @media (max-width: 355px) {
-    width: 55px;
-    position: relative;
-    right: 45px;
-  }
+  position: fixed;
+  right: 15px;
+  top: 18px;
 `
 
 export const StyledTitle = styled<{}, 'div'>('div')`
   color: #4B4C5C;
   font-size: 22px;
   font-weight: 500;
   letter-spacing: 0.12px;
-  line-height: 44px;
   margin-bottom: -23px;
+  line-height: 44px;
+
+  @media (max-width: 375px) {
+    max-width: 250px;
+    word-wrap: break-word;
+    display: block;
+    line-height: 20px;
+    margin-top: -12px;
```

Comment:
hmm, do we need all this complicated negative margins and line-height? If the aim is to have the text be vertically centered whether it's displayed on 1 line or 2, can't we have the parent be 100% height and just vertically center the text via flexbox, and then allowing wrapping at the specific width as you have here?